easybuild/easyconfigs/l/libStatGen/libStatGen-457bf9e-foss-2018a.eb

```diff
@@ -0,0 +1,32 @@
+# This easyconfig was created by Simon Branford of the BEAR Software team at the University of Birmingham.
+easyblock = 'MakeCp'
+
+name = 'libStatGen'
+version = '457bf9e'
+# hash of the current master on 30th March 2019
+fullhash = '457bf9e93d3da0e284ae35b595fbf466f38e4159'
+
+homepage = "https://genome.sph.umich.edu/wiki/C++_Library:_libStatGen"
+description = """Useful set of classes for creating statistical genetic programs."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/statgen/%(name)s/archive/']
+sources = ['%s.tar.gz' % fullhash]
```

Comment:
@branfosj Use `download_file` here to download via commit ID and rename after download?
See https://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources